lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -96,7 +96,13 @@ def word_symbol_pair?(pair)
           return false unless key.sym_type?
 
           sym_name = key.loc.expression.source
-          sym_name !~ /\A:["']|=\z/
+          valid_19_syntax_symbol?(sym_name)
+        end
+
+        def valid_19_syntax_symbol?(sym_name)
+          return false if sym_name =~ /\A:["']/
```

Comment:
Ah, that makes sense now.
When I run rake with that line removed in 2.2.1 a bunch fail, but when I run on 2.1.5 everything is green.

I can skip that line if you are on 2.2, but I'm not sure what the rubocop policy is for syntax stuff changing between ruby versions.